WolvenKit.App/Globals.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WolvenKit.App;
+public static class Globals
+{
+    public const bool ENABLE_NODE_EDITOR = false;
```

Comment:
Could this be set with a application parameter with a default of false? so that like "wolvenkit.exe --with_node_editor" or something similar can be used to enable it outside of needing to build it yourself?